README.md

```diff
@@ -57,10 +65,18 @@ $ cd cpg_flow_test
 
 # Run the test with the bash script
 $ chmod +x run-test-workflow.sh
-$ ./run-test-workflow.sh
+
+# See the notes below on how to find a valid path/tag.
+# The default path is australia-southeast1-docker.pkg.dev/cpg-common/images/cpg_flow:0.1.0-alpha.14
+$ ./run-test-workflow.sh --image "australia-southeast1-docker.pkg.dev/cpg-common/images/cpg_flow:<tag_id>"
 ```
-> **Note**
-> You will need to have `analysis-runner` installed in your environment. See the [analysis-runner](https://github.com/populationgenomics/analysis-runner) for more information or install it with `pipx install analysis-runner`.
+You should receive a job url from the `analysis-runner` output, if the job was created successfully. This job should spin up additional jobs that can be found from the `/batches` page on Hail.
+
+### Notes
+
+* Having the option to run this pipeline on different tags of the `cpg_flow` image, is valuable for testing unmerged functionality in the `cpg_flow` repository. You can always default to a recent release tag to test with a stable version of the  `cpg_flow` image.
+* You will need a valid tag above, which you can find from the most recent [`cpg-flow` docker workflow](https://github.com/populationgenomics/cpg-flow/actions/workflows/docker.yaml) runs, under the `print docker tag` job of the workflow. Be mindful of the use of `images` vs `images-tmp` which usually contains test images that are pruned fortnightly.
+* You will need to have `analysis-runner` installed in your environment. See the [analysis-runner](https://github.com/populationgenomics/analysis-runner) for more information or install it with `pipx install analysis-runner`.
```

Comment:
I think we should move this to the top, rather than a note at the bottom.

Reply:
hmm this was only meant to be a comment on the analysis-runner line, not the rest.